agents/exec/src/main/resources/org.eclipse.che.exec.script.sh

```diff
@@ -23,7 +32,7 @@ if [ ${CURL_INSTALLED} = false ] && [ ${WGET_INSTALLED} = false ]; then
   CURL_INSTALLED=true
 fi
 
-test "$(id -u)" = 0 || SUDO="sudo -E"
+is_current_user_root && is_current_user_sudoer || SUDO="sudo -E"
```

Comment:
Can you elaborate on what will happen if a user is root but sudo command is not installed in a container?